src/asset/hollowknight/categories/112-comsob-late-fury.json

```diff
@@ -137,28 +152,11 @@
             "Enter Hive Knight",
             "Enter Enraged Guardian"
         ],
-        "GodhomeLoreRoom": [
-            "Enter Watcher Knights",
-            "Enter Soul Tyrant",
-            "Enter Pure Vessel"
-        ]
-    },
-    "icons": {
-        "GodhomeBench": [
-            "GorbP",
-            "MarmuP",
-            "GalienP",
-            "FailedChampionP",
-            "XeroP",
-            "MarmuP",
-            "HiveKnightP",
-            "EnragedGuardianP"
-        ],
-        "GodhomeLoreRoom": ["WatcherKnightsP", "SoulTyrantP", "PureVesselP"]
+        "GodhomeLoreRoom": ["Enter Watcher Knights", "Enter Soul Tyrant"]
```

Comment:
Should probably be using the old overrides for GodhomeLoreRoom because this is missing Enter Pure Vessel. Also the old splits had icon overrides which were nice